src/apps/config/leader.lua

```diff
@@ -186,6 +186,18 @@ function Leader:rpc_set_alarm_operator_state (args)
    if success then return response else return {status=1, error=response} end
 end
 
+function Leader:rpc_purge_alarms (args)
+   local function getter()
```

Comment:
Can we rename this function, perhaps to `purge`?

Reply:
I can do it but I think is clearer what it does as `purge_alarms` rather than `purge` alone.